src/styles/home_body.css

```diff
@@ -0,0 +1,54 @@
+@media only screen and (max-width: 720px) {
+  .home-main {
+    width: 100%;
+    margin: auto;
+  }
+}
+
+@media only screen and (max-width: 1800px) {
+  .home-main {
+    width: 100%;
+    margin: auto;
+  }
+}
+
+.home-page {
+  font-family: 'Questrial', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  overflow: hidden;
+}
+
+.home-page-logo {
+  animation: home-page-logo-spin infinite 20s linear;
+  height: 80px;
+}
+
+.home-page-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.home-page-intro {
+  font-size: large;
+}
+
+@keyframes home-page-logo-spin {
+  from { transform: rotate(0deg); }
```

Comment:
No single line rules please.
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```